### PR TITLE
Update token logo urls

### PR DESCRIPTION
**Pull request by hungdoansy**
# Description
Update token urls that are no longer existent

### CODE REVIEW

File: tokenLists/fantom.mainnet.tokenlist.json [hunk 1]
```diff
@@ -284,16 +280,15 @@
...
         {
             "name": "aUSD",
             "symbol": "aUSD",
             "address": "0x41e3dF7f716aB5AF28c1497B354D79342923196a",
             "_scan": "https://ftmscan.com/token/0x41e3dF7f716aB5AF28c1497B354D79342923196a",
             "chainId": 250,
-            "decimals": 18,
-            "logoURI": "https://aquarius.fi/assets/200/AUSD200-t.png"
+            "decimals": 18
         },
```
Comment (viet-nv) [inline tokenLists/fantom.mainnet.tokenlist.json]: why do you remove logo of this token?
I see aquarius link is error but you can update this to our source then user raw github link 
image

File: tokenLists/fantom.mainnet.tokenlist.json [hunk 2]
```diff
@@ -347,16 +342,15 @@
...
         {
             "name": "iFUSD",
             "symbol": "iFUSD",
             "address": "0x9fC071cE771c7B27b7d9A57C32c0a84c18200F8a",
             "_scan": "https://ftmscan.com/token/0x9fC071cE771c7B27b7d9A57C32c0a84c18200F8a",
             "chainId": 250,
-            "decimals": 18,
-            "logoURI": "https://assets.spookyswap.finance/tokens/iFUSD.png"
+            "decimals": 18
```
Comment (viet-nv) [inline tokenLists/fantom.mainnet.tokenlist.json:355]: same for this, add logo for it

File: tokenLists/fantom.mainnet.tokenlist.json [hunk 3]
```diff
@@ -392,16 +386,15 @@
...
         },
         {
             "name": "The Eye of Truth",
             "symbol": "EYE",
             "address": "0x496e1693A7B162c4f0Cd6a1792679cC48EcbCC8d",
             "_scan": "https://ftmscan.com/token/0x496e1693A7B162c4f0Cd6a1792679cC48EcbCC8d",
             "chainId": 250,
-            "decimals": 18,
-            "logoURI": "https://assets.spookyswap.finance/tokens/EYE.png"
+            "decimals": 18
```
Comment (viet-nv) [inline tokenLists/fantom.mainnet.tokenlist.json:399]: add logo

File: tokenLists/fantom.mainnet.tokenlist.json [hunk 4]
```diff
@@ -431,8 +422,7 @@
             "address": "0xFbc3c04845162F067A0B6F8934383E63899c3524",
             "_scan": "https://ftmscan.com/token/0xFbc3c04845162F067A0B6F8934383E63899c3524",
             "chainId": 250,
-            "decimals": 18,
-            "logoURI": "https://assets.spookyswap.finance/tokens/FOO.png"
+            "decimals": 18
```
Comment (viet-nv) [inline tokenLists/fantom.mainnet.tokenlist.json:427]: add logo

File: tokenLists/fantom.mainnet.tokenlist.json [hunk 5]
```diff
@@ -421,7 +413,6 @@
             "_scan": "https://ftmscan.com/token/0x47d0d625638b56084e76b8720475d175d171af9A",
             "chainId": 250,
             "decimals": 18,
```
Comment (viet-nv) [inline tokenLists/fantom.mainnet.tokenlist.json:417]: add logo

File: tokenLists/fantom.mainnet.tokenlist.json [hunk 6]
```diff
@@ -522,16 +512,15 @@
...
         },
         {
             "name": "Iron Finance ICE Token",
             "symbol": "IronICE",
             "address": "0x260b3E40c714Ce8196465Ec824Cd8Bb915081812",
             "_scan": "https://ftmscan.com/token/0x260b3E40c714Ce8196465Ec824Cd8Bb915081812",
             "chainId": 250,
-            "decimals": 18,
-            "logoURI": "https://assets.spookyswap.finance/tokens/IronICE.png"
+            "decimals": 18
```
Comment (viet-nv) [inline tokenLists/fantom.mainnet.tokenlist.json:525]: add logo

File: tokenLists/fantom.mainnet.tokenlist.json [hunk 7]
```diff
@@ -576,16 +565,15 @@
...
         },
         {
             "name": "Sing Token",
             "symbol": "SING",
             "address": "0x53D831e1db0947c74e8a52618f662209ec5dE0cE",
             "_scan": "https://ftmscan.com/token/0x53D831e1db0947c74e8a52618f662209ec5dE0cE",
             "chainId": 250,
-            "decimals": 18,
-            "logoURI": "https://assets.spookyswap.finance/tokens/SING.png"
+            "decimals": 18
```
Comment (viet-nv) [inline tokenLists/fantom.mainnet.tokenlist.json:578]: add logo

File: tokenLists/fantom.mainnet.tokenlist.json [hunk 8]
```diff
@@ -672,7 +660,6 @@
             "_scan": "https://ftmscan.com/token/0xb84527D59b6Ecb96F433029ECc890D4492C5dCe1",
             "chainId": 250,
             "decimals": 18,
```
Comment (viet-nv) [inline tokenLists/fantom.mainnet.tokenlist.json:664]: add logo

File: tokenLists/fantom.mainnet.tokenlist.json [hunk 9]
```diff
@@ -817,7 +804,6 @@
             "_scan": "https://ftmscan.com/token/0x9F47F313ACFd4bdC52F4373b493EaE7d5aC5b765",
             "chainId": 250,
             "decimals": 18,
```
Comment (viet-nv) [inline tokenLists/fantom.mainnet.tokenlist.json:808]: add logo

File: tokenLists/oasis.tokenlist.json [hunk 10]
```diff
@@ -58,8 +55,7 @@
             "_scan": "https://explorer.emerald.oasis.dev/token/0x0bfF36Be5cf671Fa973f8206483b6641A90CE7d0",
             "symbol": "🌹SWAP",
             "name": "🌹RoseSwap.io",
-            "decimals": 18,
-            "logoURI": "https://app.rose.fi/static/media/rose.54bf1c83.svg"
+            "decimals": 18
```
Comment (viet-nv) [inline tokenLists/oasis.tokenlist.json]: add logo

**Review by viet-nv — CHANGES_REQUESTED** [review]
A lot of token logo have been removed. Please add new image if old link is error

File: tokenLists/fantom.mainnet.tokenlist.json (hunk 1 above)
Comment (hungdoansy) [reply tokenLists/fantom.mainnet.tokenlist.json]: Updated

File: tokenLists/fantom.mainnet.tokenlist.json (hunk 3 above)
Comment (hungdoansy) [reply tokenLists/fantom.mainnet.tokenlist.json:399]: The URL is no longer accessible and I couldn't find the logo URL in CoinMarketCap and ftmscan.com, so I removed it

File: tokenLists/fantom.mainnet.tokenlist.json (hunk 4 above)
Comment (hungdoansy) [reply tokenLists/fantom.mainnet.tokenlist.json:427]: The URL is no longer accessible and I couldn't find the logo URL in CoinMarketCap and ftmscan.com, so I removed it

File: tokenLists/fantom.mainnet.tokenlist.json (hunk 5 above)
Comment (hungdoansy) [reply tokenLists/fantom.mainnet.tokenlist.json:417]: The URL is no longer accessible and I couldn't find the logo URL in CoinMarketCap and ftmscan.com, so I removed it

File: tokenLists/fantom.mainnet.tokenlist.json (hunk 6 above)
Comment (hungdoansy) [reply tokenLists/fantom.mainnet.tokenlist.json:525]: The URL is no longer accessible and I couldn't find the logo URL in CoinMarketCap and ftmscan.com, so I removed it

File: tokenLists/fantom.mainnet.tokenlist.json (hunk 7 above)
Comment (hungdoansy) [reply tokenLists/fantom.mainnet.tokenlist.json:578]: The URL is no longer accessible and I couldn't find the logo URL in CoinMarketCap and ftmscan.com, so I removed it

File: tokenLists/oasis.tokenlist.json (hunk 10 above)
Comment (hungdoansy) [reply tokenLists/oasis.tokenlist.json]: Updated

**Review by namgold — COMMENTED** [review]
Please change the token img name folow the convension `img/token/aUSD.png`

**Comment by hungdoansy** [comment]
> Please change the token img name folow the convension `img/token/aUSD.png`

@viet-nv suggested and I agree with him that we should split them into chains

**Review by viet-nv — APPROVED** [review]

**Review by namgold — APPROVED** [review]

**Pull request merged**